app/code/Magento/CatalogImportExport/Model/Import/Product/Validator.php

```diff
@@ -71,11 +71,11 @@ protected function textValidation($attrCode, $type)
     {
         $val = $this->string->cleanString($this->_rowData[$attrCode]);
```

Comment:
How about extracting that to separate method `isTextValid` that will return the result immediately?

There will be no need to use elseif / else, then. The result of isTextValid will be assigned to $valid (I'd rename that to $isValid)